librz/core/cmd/cmd_open.c

```diff
@@ -1588,3 +1567,25 @@ RZ_IPI RzCmdStatus rz_open_close_all_handler(RzCore *core, int argc, const char
 	RZ_LOG_INFO("Close all files\n");
 	return RZ_CMD_STATUS_OK;
 }
+
+RZ_IPI RzCmdStatus rz_open_use_handler(RzCore *core, int argc, const char **argv) {
+	RzListIter *iter = NULL;
+	RzCoreFile *f;
+
+	int fdnum = rz_num_math(NULL, argv[1]);
+	rz_list_foreach (core->files, iter, f) {
+		if (f->fd == fdnum) {
+			core->file = f;
+			rz_io_use_fd(core->io, fdnum);
+			RzBinFile *bf = rz_bin_file_find_by_fd(core->bin, fdnum);
+			if (bf) {
+				rz_core_bin_raise(core, bf->id);
+				rz_core_block_read(core);
+				RZ_LOG_INFO("Switched to fd %d (%s)\n", fdnum, bf->file);
+			}
+			return RZ_CMD_STATUS_OK;
+		}
+	}
+	RZ_LOG_ERROR("Could not find any opened file with fd %d\n", fdnum);
+	return RZ_CMD_STATUS_ERROR;
```

Comment:
this is just a comment, but this code does not really work. we need to fix this in another commit.

Reply:
what do you mean? What does not work?

Reply:
try it. just does not really work. print a function, change fd and then try again with another function

Reply:
ah if you mean analysis I didn't test it. It should work though at a more low level.